### ISSUE TITLE
[JetBrains Toolbox Search Recent Projects] Not finding remote projects (Gateway)

### ISSUE THREAD

**Issue by delucca** [raycast/extensions#15053]
### Extension

https://www.raycast.com/gdsmith/jetbrains

### Raycast Version

1.84.3

### macOS Version

15.0.1

### Description

The extension works fine for local projects, but it can't fetch any projects from JetBrains Gateway and/or projects on SSH remote machines (that are configured and I can see as recent projects on the JetBrains app

### Steps To Reproduce

1. Setup a remote project either using JetBrains Gateway, or with a SSH connection on any JetBrains app
2. Try to access it with the extension

### Current Behaviour

The extension doesn't list remote projects

### Expected Behaviour

The extension should list remote projects

**Comment by gdsmith** [comment]
@raycastbot close as not planned

Both JetBrains Gateway and Remote Development are beta features so it's unlikely support will be added until/unless someone with the desire to understand the changes that would be needed to support those features.

**Comment by ChillarAnand** [comment]
Now that these features are out of beta and fully supported, do you plan to add this now?

**Comment by gdsmith** [comment]
I've looked at it and the setup and configuration are complex enough to make me not want to use it (that's without considering the JB agent running on the target machine with some pretty hefty memory/cpu requirements). It does seem like remote development is still in beta, even if the Gateway.app itself is not.

I did attempt to figure out where the configuration is stored to grab the necessary information, but I couldn't figure out enough about the structure or even what command would need to be run to open the remote projects even after finding the required information. Any help figuring that out would greatly improve the likelihood of this getting implemented.
